odoo/addons/base/res/res_country_view.xml

```diff
@@ -110,6 +110,9 @@
             <field name="model">res.country.state</field>
             <field name="arch" type="xml">
                 <form string="State">
+                    <div class="oe_button_box">
```

Comment:
don't want to add an empty div because people may add things in it, can be done via an xpath if needed